packages/node_modules/sample-browser-auth-implicit/test/wdio/specs/implicit-grant-authentication.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef */
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import {assert} from 'chai';
+
+describe(`sample-browser-auth-implicit`, () => {
+  describe(`authenticate using implicit grant flow`, () => {
+    const browserSpock = browser.select(`browserSpock`);
+    let spock;
+
+    before(`create Spock`, () => testUsers.create({count: 1, config: {displayName: `Spock`}})
+      .then((users) => {
+        [spock] = users;
+      }));
+
+
+    step(`browse to sample app, and verify the user is not authenticated`, () => {
+      browserSpock.url(`/packages/node_modules/sample-browser-auth-implicit`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Not Authenticated`);
+    });
+
+    step(`initiate implicit grant authentication flow`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+    });
+
+    step(`enter user login information`, () => {
+      // Enter Email
+      browserSpock.waitUntil(() => browser.select(`browserSpock`).getTitle() === `Sign In - Cisco Spark`);
+      browserSpock.setValue(`[placeholder="Email address"]`, spock.email);
+      browserSpock.click(`[name="btnOK"]`);
+
+      // Enter Password
+      browserSpock.waitUntil(() => browserSpock.getTitle() === `Sign In - Cisco Spark`);
```

Comment:
Aren't we already at a page with this title? There's a good chance to could return true before the password field is ready, right? would it be more useful to wait for the password field to be visible?